src/journal/models.py

```diff
@@ -239,9 +239,17 @@ def editor_pks(self):
 
     def journal_users(self, objects=True):
         if objects:
-            users = [role.user for role in core_models.AccountRole.objects.filter(journal=self, user__is_active=True)]
+            users = [role.user for role in
+                     core_models.AccountRole.objects.filter(
+                         journal=self,
+                         user__is_active=True,
+                     ).select_related('user')]
         else:
-            users = [role.user.pk for role in core_models.AccountRole.objects.filter(journal=self, user__is_active=True)]
+            users = [role.user.pk for role in
```

Comment:
could do as above and avoid constructing intermediate list as it is not used for anything

Reply:
Do you mean return a set rather than a list?

Reply:
It is returning a set already, but constructing an intermediate list that serves no purpose, same as https://github.com/BirkbeckCTP/janeway/pull/1046#discussion_r269079711